op-acceptance-tests/tests/interop/proofs/proposer_test.go

```diff
@@ -0,0 +1,61 @@
+package proofs
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lmittmann/w3"
+)
+
+var (
+	gameCountFn        = w3.MustNewFunc("gameCount()", "uint256")
+	gameAtIndexFn      = w3.MustNewFunc("gameAtIndex(uint256)", "uint32, uint64, address")
+	rootClaimFn        = w3.MustNewFunc("rootClaim()", "bytes32")
+	l2SequenceNumberFn = w3.MustNewFunc("l2SequenceNumber()", "uint256")
+)
+
+func TestProposer(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewSimpleInterop(t)
+
+	deployment := sys.L2Networks()[0].Escape().Deployment()
```

Comment:
```suggestion
	deployment := sys.L2ChainA.Deployment()
```

(nit) Might be nicer to expose the addresses directly with something like: `sys.L2ChainA.DisputeGameFactoryProxyAddr()`

Reply:
good suggestion. Fixed.